docs/spec/overview.md

```diff
@@ -1,18 +1,130 @@
 
-# Overview
-
-1. [Introduction](./introduction.md)
-2. [Glossary](./definitions.md)
-3. [Data Model](./data-model.md)
-4. [Architecture](./architecture.md)
-5. Flows
-    - [Dispersal](./flows/dispersal.md)
-    - Retrieval
-    - Payment
-6. Protocol Modules 
-    - [Attestation](./protocol-modules/attestation/attestation.md)
-    - [Storage](./protocol-modules/storage/overview.md)
-    - [Retrieval](./protocol-modules/retrieval//retrieval.md)
-    - Payment
-7. System Properties
-    - [Throughput](./throughput.md)
\ No newline at end of file
+
+## Introduction
+
+
+EigenDA is a Data Availability (DA) service, implemented as an actively validated service (AVS) on EigenLayer, that provides secure and scalable DA for L2s on Ethereum. 
+
+### What is DA?
+
+In informal terms, DA is a guarantee that a given piece of data will be available to anyone who wishes to retrieve it. 
+
+A DA system accepts blobs of data (via some interface) and then makes them available to retrievers (through another interface). 
+
+Two important aspects of a DA system are 
+1. Security: The security of a DA system constitutes the set of conditions which are sufficient to ensure that all data blobs certified by the system as available are indeed available for honest retrievers to download. 
+2. Throughput: The throughput of a DA system is the rate at which the system is able to accept blobs of data, typically measured in bytes/second. 
+
+### An EigenLayer AVS for DA
+
+EigenDA is implemented as an actively validated service on EigenLayer, which is a restaking protocol for Ethereum. 
+
+Beause of this, EigenDA makes use of the EigenLayer state, which is stored on Ethereum, for consensus about the state of operators and as a callback for consensus about the availability of data. This means that EigenDA can be simpler in implementation than many existing DA solutions: EigenDA doesn't need to build it's own chain; it rides on the back of Ethereum. 
+
+### A first of its kind, horizontally scalable DA solution
+
+Among extant DA solutions, EigenDA takes an approach to scalability which is unique in that it yields true horizontal scalability: Every additional unit of capacity contributed by a operator can increase the total system capacity. 
+
+This property is achieved by using a Reed Solomon erasure encoding scheme to shard the blob data across the DA nodes. While other systems such as Celestia and Danksharding (planned) also make use of Reed Solomon encoding, they do so only for the purpose of supporting certain observability properties of Data Availability Sampling (DAS) by light nodes. On the other hand, all incentivized/full nodes of the system download, store, and serve the full system bandwidth. 
+
+Horizontal scalability provides the promise for the technological bottlnecks of DA capacity to continually track demand, which has enormous implications for Layer 2 ecosytems. 
```

Comment:
"bottlnecks" typo and maybe the sentence is a bit off.

Would be useful to run a grammar checker on the text (otherwise we may receive many doc fix PRs)